### DISCUSSION TITLE
Crash using react-navigation with react-native-screens version 3.1.0

### DISCUSSION THREAD

**Discussion by prupipho** [software-mansion/react-native-screens#884] [Q&A] [answered]
Could this issue [folly::toJson: JSON object value was a NaN or INF error on rendering simple navigation examples
 || Invariant Violation: requireNativeComponent: "RNSScreen" was not found](https://github.com/react-navigation/react-navigation/issues/9480) be related to this repo instead of the react-navigation one?

**Comment by kacperkapusciak** [comment]
Hi @prupipho,
"Invariant Violation: requireNativeComponent: "RNSScreen" was not found " looks like a misconfiguration of development environment but it is definitely connected to `react-native-screens`.
On the other hand "folly::toJson: JSON object value was a NaN or INF" doesn't look like something that we could help with.

Do you mind submitting an issue here with a repo/snack with the minimal needed configuration?

**Reply by prupipho** [reply]
Thanks. I will do that.

**Reply by WoLewicki** [reply, marked as answer]
I believe it was fixed in https://github.com/software-mansion/react-native-screens/issues/885 and released in `3.1.1` so we can mark it as answered.

**Reply by prupipho** [reply]
@WoLewicki Do I need to do something? I had a quick look to see how I can mark it as answered but I couldn´t find the option to do it.

**Reply by WoLewicki** [reply]
Hmm looks like this category of discussion does not have this option. I will change it and try to set it to answered.

**Reply by WoLewicki** [reply]
Done.

**Reply by prupipho** [reply]
Thanks :)
